### ISSUE TITLE
Example of platform filtering is missing a section

### ISSUE THREAD

**Issue by ynlng** [pypa/bandersnatch#265]
In file filtering_configuration.md, the example of Platform-specific binaries filtering is missing the "[blacklist] section.

It should be:
```
[plugins]
enabled =
    exclude_platform
[blacklist]
platforms =
    macos
```

**Comment by cooperlees** [comment]
Thanks for hi-lighting. PR welcome. Will try get to it when I can.

**Comment by cooperlees** [comment]
Fixed by #266. TGhanks @ynlng !
